docs/spec.md

```diff
@@ -1923,6 +1923,55 @@ imaginary values, `lhs` and `rhs`, and produces a `result` tensor.
 
 &nbsp;[More Examples](../stablehlo/tests/interpret/complex.mlir)
 
+### composite
+
+#### Semantics
+
+Encapsulates an operation made up (composed) of other StableHLO operations,
+taking `inputs` and producing `results`. The decomposition is specified
+using the `decomposition` attribute. The composite op can be replaced with
+its decomposition without changing semantics.
```

Comment:
How does one test this in practicality?
You'd always want the inlined version to test against?

Reply:
I don't think this necessarily needs to be tested, although maybe once we have the inlining pass we can add a test that inlines and then runs the interpreter to validate this.